Automattic/vip-go-mu-plugins
# Tracks indexing job lenght

## Description

First step towards storing statsD metrics around VIP search. This PR adds indexing time metric under:
`stats.timers.com.wordpress.elasticsearch.$dc.[[node]]_[[cluster]].bulk_index.[total|engine]` key

## Checklist

Please make sure the items below have been covered before requesting a review:


## Steps to Test

Outline the steps to test and verify the PR here.

Example:

1. in local environement run ` lando wp vip-search index` (with --setup if ran for the first time)
1. Then you can do `lando logs -s stats` to see the recorded stats
